lib/event_sourcery/postgres/tracker.rb

```diff
@@ -1,6 +1,8 @@
 module EventSourcery
   module Postgres
     class Tracker
+      # This will set up a persisted event id tracker for processors.
```

Comment:
Is this describing the `Tracker` class? If so it should be above the `class Tracker` line.